charon/constants.py

```diff
@@ -173,3 +173,5 @@
 PROD_INFO_SUFFIX = ".prodinfo"
 MANIFEST_SUFFIX = ".txt"
 DEFAULT_ERRORS_LOG = "errors.log"
+
+NRRC_REGISTRY = "npm.registry.redhat.com"
```

Comment:
I think we should leave this out as a configuration item below each "target", as it is not same in different environments. We can set this as DEFAULT_NRRC_REGISTRY with a default value(for example "localhost" ) if it is not specified in configuration,

Reply:
@ligangty If for multi targets uploading, and configuring different registries with the targets, it has to handle the different metadata content-generating for different target/env, I want to confirm there will be any such case we will encounter in prod/stage?

Reply:
Yes there will be such case. We will not prevent user to upload one tarball to both prod/stage in one time.